jdaviz/utils.py

```diff
@@ -355,6 +355,12 @@ def flux_conversion(spec, values, original_units, target_units):
     return (values * orig_units).to_value(targ_units, equivalencies=eqv)
 
 
+def _convert_surface_brightness_units(data, from_unit, to_unit):
```

Comment:
My remaining question is whether this PR covers converting from per-wavelength flux unit to/from per-frequency flux unit. Because I don't see `u.spectral_density(...)` being used.